requirements-docs.txt

```diff
@@ -15,7 +15,7 @@ livereload==2.6.3
 markdown==3.6
 markupsafe==2.1.5
 mergedeep==1.3.4
-mkdocs==1.6.0
+mkdocs==1.3.0
```

Comment:
Why is this dep. being downgraded?

Reply:
I think this happened as I was running these commands.
![image](https://github.com/user-attachments/assets/0e30e363-c3d0-4b76-9850-91d584decea7)


Look at this commit: ecc016d6b004b202c26f0f2a3ef9debbd8606301

Reply:
Do you want me to upgrade it back?

Reply:
Yes, revert every dependency change. Since they are not part (nor required) of this PR.
That image shows a list of dev commands, but not the order in which they need to be done (not really anyway).
Feel free to update the docs on that if they make it easier for new joiners.

Reply:
Don't really know the "right" order for the commands. What I have done is that I ran the commands in the original order as per the docs, and then I reverted the dependency changes. It worked this way.